applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -482,3 +587,124 @@ int cloud_encode_light_sensor_data(const struct light_sensor_data *sensor_data,
 	return cloud_encode_data(&cloud_sensor, output);
 }
 #endif /* CONFIG_LIGHT_SENSOR */
+
+static int sensor_ch_cfg_set_item(struct sensor_ch_cfg *const cfg,
+				  const enum sensor_ch_cfg_item_type type,
+				  const double value)
+{
+	if ((type < SENSOR_CH_CFG_ITEM_TYPE__BEGIN) ||
+	    (type >= SENSOR_CH_CFG_ITEM_TYPE__END) || (cfg == NULL)) {
+		return -EINVAL;
+	}
+
+	cfg->value[type] = value;
+
+	return 0;
+}
+
+static bool sensor_ch_cfg_is_send_allowed(const struct sensor_ch_cfg *const cfg,
```

Comment:
Function name reads like a question to me. Not necessarily a bad thing, though, just not used to it :)